examples/walkdir.rs

```diff
@@ -0,0 +1,10 @@
+extern crate walkdir;
+
+fn main(){
+use walkdir::WalkDir;
+
+for entry in WalkDir::new("directory_name") {
+    let entry = entry.unwrap();
+    println!("{}", entry.path().display());
+   }
+}
```

Comment:
Indentation is incorrect in this function.

I'd prefer not to have `unwrap` in these examples. Can you change this to use correct error handling? Since there's only one error type possible here we don't have to use error_chain, but can use a simple scheme like the one already used in the [flate2](https://github.com/brson/stdx#flate2--0219--) example.

So replace the `main` definition with `fn run() -> Result<(), WhateverErrorTypeWalkDirUses>`, and end the example with `fn main() { run().unwrap() }`.